src/main/resources/portal.properties.EXAMPLE

```diff
@@ -119,6 +119,8 @@ ldap.attributes.given_name=givenName
 ldap.attributes.email=mail
 
 # data access token settings
+dat.unauth_users=
```

Comment:
I think the property name is not clear. A self-documenting name such as "dat.ineligible_user_list" would be better, or we could give a comment above the property explaining what goes into the field : (comma/whitespace?) separated list  of users who will be refused access to the Data Access Token methods

Reply:
@sheridancbio I added some detailed descriptions of all of these properties in the documentation PR (#5306)